lib/publish.js

```diff
@@ -154,10 +155,29 @@ class Publish extends BaseCommand {
     const results = {}
     const json = this.npm.config.get('json')
     const silent = log.level === 'silent'
+    const noop = a => a
+    const color = this.npm.color ? chalk : { green: noop, bold: noop }
```

Comment:
One of these days we should pull this repeated logic into npm.js itself.

Reply:
👍 either that or maybe it ought to live in one of the base commands parent classes to be reused